.github/workflows/release.yml

```diff
@@ -1,8 +1,8 @@
 name: Release Coffeecard App
 
 on:
-  push:
-    branches: [develop, production]
+  pull_request:
+    branches: [develop, production, feature/omid/add-debug-build]
```

Comment:
Is it intentional that push has been changed to pull_request?
I assume adding your branch here is for debugging purposes, remember to remove

Reply:
Same as above